source/extract_combine_patches.ipynb

```diff
@@ -0,0 +1,323 @@
+{
```

Comment:
<p>Would be great too visualise the images, then ready</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kornia/tutorials/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Currently I'm using a random array so that the notebook is lightweight. Do you have example image or a notebook that does something similar?

Reply:
Actually what would be the point of visualising images? Once you extract and combine you will get the same image which is already shown via `assert (image == restored_img).all()`.

Reply:
Showing images always makes tutorials way better. Choose an image of your interest and download with whet, see here https://github.com/kornia/tutorials/blob/master/source/canny.ipynb

Reply:
Done

Reply:
Very nice! Love it